src/Browscap/Data/Platform.php

```diff
@@ -55,33 +42,33 @@ public function __construct(string $match, array $properties, bool $isLite, bool
     }
 
     /**
-     * @return string
+     * @return string|null
```

Comment:
Unnecessary phpdocs here; `@return` type is already declared, so don't need docblocks on any of the methods here

Reply:
changed